### PR TITLE
fix: Fixes parsing of boolean environment variables

### PR DESCRIPTION
**Pull request by dappelt**
If a environment varialbe was set to the literal false, it was treated as being set to true.

### CODE REVIEW

File: bin/normalizeEnv.js [hunk 1]
```diff
@@ -8,10 +8,22 @@ const generateSecret = (secret, name) => {
...
   return process.env[name] || envVars[name] || def
 }
 
+const parseBool = (name, def = false) => {
+  const val = getVar(name).toLowerCase()
+  if (val === 'true') {
+    return true
+  } else if (val === 'false') {
+    return false
+  } else if (val === '') { // if the varialbe is undefined
```
Comment (sharafian) [inline bin/normalizeEnv.js]: varialbe -> variable

File: bin/normalizeEnv.js [hunk 2]
```diff
@@ -8,10 +8,22 @@ const generateSecret = (secret, name) => {
...
+    return def
+  }
+  throw {_error: 'Can not parse as boolean: ' + name}
```
Comment (sharafian) [inline bin/normalizeEnv.js]: This should trigger `process.exit(1)`, like it does [here](https://github.com/interledgerjs/ilp-kit/pull/168/files#diff-3641b3524271ca689c53abb4d4a7e8a4R55)

**Review by sharafian — CHANGES_REQUESTED** [review]

**Comment by vhpoet** [comment]
@dappelt five-bells-shared already has code for this https://github.com/interledgerjs/five-bells-shared/blob/master/lib/config.js#L42-L63

**Review by vhpoet — CHANGES_REQUESTED** [review]
@dappelt five-bells-shared already has code for this https://github.com/interledgerjs/five-bells-shared/blob/master/lib/config.js#L42-L63, please use that instead.

**Comment by dappelt** [comment]
done

**Review by sharafian — APPROVED** [review]

**Review by vhpoet — APPROVED** [review]

**Pull request merged**